plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftClient.java

```diff
@@ -256,6 +255,7 @@ public RedshiftClient(
                 connectorExpressionRewriter,
                 ImmutableSet.<ProjectFunctionRule<JdbcExpression, ParameterizedExpression>>builder()
                         .add(new RewriteCast((session, type) -> toWriteMapping(session, type).getDataType()))
+                        .add(new RewriteLpad())
```

Comment:
> Pushdown lpad string function in Redshift

Could you include "projection" in a commit title? "Add support for projection pushdown with lpad function in Redshift" or something.